sdk/storage/azblob/constants.go

```diff
@@ -6,33 +6,49 @@
 
 package azblob
 
+var SASVersion = "2019-12-12"
```

Comment:
should this be a constant?

Reply:
I would assume so given that this is defined in `constants.go`.  Unless it's intended to be changed by the caller.  @mohsha-msft can you comment?

Reply:
Yes, having the SASVersion as var can give caller the option to change it if needed without making change in the code.